recipes/pymove/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "pymove" %}
+{% set version = "1.1.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b6d8dfbe9a6b7f0cb8587a70db5bfc385c0a9395a5e430a61f38ec45198e5b77
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - branca
+    - dask
+    - folium >=0.10.1
+    - geojson
+    - holidays
+    - ipywidgets
+    - joblib
+    - matplotlib
+    - mplleaflet
+    - numpy
+    - pandas
+    - pip
+    - psutil
+    - python
+    - pytz ==2019.3
+    - scikit-learn
+    - scipy
+    - shapely
+    - tqdm
+  run:
+    - branca
+    - dask
+    - folium >=0.10.1
+    - geojson
+    - holidays
+    - ipywidgets
+    - joblib
+    - matplotlib
+    - mplleaflet
+    - numpy
+    - pandas
+    - psutil
+    - python
+    - pytz ==2019.3
+    - scikit-learn
+    - scipy
+    - shapely
+    - tqdm
+
+test:
+  imports:
+    - pymove
+    - pymove.core
+    - pymove.models
+    - pymove.models.pattern_mining
+    - pymove.osm_module
+    - pymove.preprocessing
+    - pymove.uncertainty
+    - pymove.utils
+    - pymove.visualization
+
+about:
+  home: "https://github.com/InsightLab/PyMove"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
Add a comment above the `license_file` line that includes a link to the following PR (https://github.com/InsightLab/PyMove/pull/66), which would, if merged, allow you to remove the manually packaged copy of the license file and use one that comes in the source distribution. The comment is meant as a reminder to check for this change in future updates to the feedstock.